10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
-
-### Library Features Table
-| Library Features| Index |
-|---|---|
-| HTO-1  | HTO-Index-1 |
-| CD8_HIMC-1_Lot-1  | ADT-Index-1 |  
+- `FASTQs`: name of the "set of FASTQs" that will be fed to a single `cellranger count` run.
+  - Composed of the `Loading Sample` name appended with the `BCL Run ID`.
+  - Allows us to handle the common scenario where the same sequencing library (e.g. tube of liquid that can be aliquoted from) is sequenced more than once.
+- `Loading Sample`: name of the sample that is loaded into a 10x chip lane (can consist of several samples via hashing).
+- `Hashed Sample`: True/False, indicates whether hashing has been done (will be redundant with the `Loading Sample` naming convention).
+- `10x Lane ID`: lane number a sample is loaded into
+  - necessary for situations where the same sample is loaded into several lanes
+  - (becomes part of the FASTQs' names in that case)
+- `Library Type`: type of library being prepared (e.g. GEX, ADT)
+  - Chemistry and version may or may not be included, e.g. GEX_5-prime.
+- `BCL Run ID`: name of the BCL file the FASTQs will be put into.
+- `Processing Run`: name of the ["processing run"][`Processing-Run`] that the data is being organized under (a processing run consists of the jobs necessary to convert BCL inputs into FBM and TCR/VDJ outputs).
+
+## Sample-Level Spreadsheet
+| Sample Name  | Loading Sample |  Sample Metadata ... | HTO | Expected Cell Number | Library |
+|---|---|---|---|---|---|
+| S1_GEX  | H1_GEX | ... | HTO-1  | 5000 | L-1 |
+
+<!-- Difference b/t Sample name and Loading sample? -->
+A CITE-seq sequencing-library will, in general, contain ADTs and HTOs. <!-- Should there be an ADTs column? -->
+- `Sample Metadata`: will include necessary Cell Ranger information like organism, reference transcriptome, etc.
+- `Library`: link to the [Library Features Table] below.
+<!-- other column descriptions? -->
+
+## Library Features Table
+| Library | Feature | Index |
+|---|---|---|
+| L-1 | HTO-1 <!-- is this supposed to match the HTO column above? -->  | HTO-Index-1 |
+| L-1 | CD8_HIMC-1_Lot-1  | ADT-Index-1 |
+| … | …  | … |
+| L-2 | CD3_HIMC-1_Lot-1  | ADT-Index-2 |
+| L-2 | CD4_HIMC-1_Lot-1  | ADT-Index-3 |
 
-The Sample-Level Spreasheet above has links to Library Feature Tables - for convenience are stored as paired-columns rather than rows. The Library Features column will contain a list of feature names. The number of features varies from library to library so this spreadsheet is expected to have pairs of columns of varying lengths.
+Each library has a set of feature/index pairs associated with it, and these mappings are stored in the Library Features Table.
```

Comment:
just highlighting that I effectively performed a join / normalized the schema here, per the note in the PR description.

Reply:
ok, not totally clear to me but we can adjust if needed.

Reply:
ok, this will be good to get aligned on. I'll try to come up with some more explicit examples